Projects/SEC/SEC01/SEC02/SEC02-AZ100.md

```diff
@@ -0,0 +1,32 @@
+# SEC02-AWS100 — Use a managed Config Rule
+
+## Cloud Service Provider
+- Amazon Web Services
+
+## Difficulty
+
+- Level 100 (Introductory)
+
+## Project's Author(s)
+
+- [Andrew Brown](https://twitter.com/andrewbrown)
+
+## Objectives
+
+### You need to complete the following:
+
+- Turn on AWS Config in the US-EAST-1 region
+- Choose the managed Config rule eg. encrypted-volumes
+- Launch an EC2 instance without an encryped EBS volume
+- Monitor AWS Config until it detects there is an EBS volume that is unencrpyted
+
+### You need to answer the following:
+
+- What is AWS Config?
+- What does an AWS Config rule do?
+- What is redmedation in the context of a config rule?
```

Comment:
- Typo: remediation (I think?)

- Filename incorrect: AZ100 instead of AWS100

Reply:
Yes to all the things @what-name said.